src/artemis/device_setup_plans/setup_zebra_for_fgs.py

```diff
@@ -0,0 +1,25 @@
+from bluesky.plan_stubs import abs_set
+
+from artemis.devices.zebra import (
+    DISCONNECT,
+    IN3_TTL,
+    IN4_TTL,
+    OR1,
+    PC_PULSE,
+    TTL_DETECTOR,
+    TTL_SHUTTER,
+    TTL_XSPRESS3,
+    Zebra,
+)
+
+
+def setup_zebra_for_fgs(zebra: Zebra):
+    yield from abs_set(zebra.output.out_pvs[TTL_DETECTOR], IN3_TTL)
+    yield from abs_set(zebra.output.out_pvs[TTL_SHUTTER], IN4_TTL)
+    yield from abs_set(zebra.output.out_pvs[TTL_XSPRESS3], DISCONNECT)
+    yield from abs_set(zebra.output.pulse_1_input, DISCONNECT)
```

Comment:
Nit: I know this is an extension but it would be nice to put these all in a `group` that we can wait on later now that it's easy to do so.

Reply:
done